client/src/components/ReportCollection.js

```diff
@@ -212,6 +218,44 @@ export default class ReportCollection extends Component {
     )
   }
 
+  getEvents = reports => {
+    if (reports) {
+      return reports.map(r => {
+        const who =
+          (r.primaryAdvisor && new Person(r.primaryAdvisor).toString()) || ""
+        const where =
+          (r.principalOrg && r.principalOrg.shortName) ||
+          (r.location && r.location.name) ||
+          ""
+
+        return {
+          title: who + "@" + where,
+          start: moment(r.engagementDate).format("YYYY-MM-DD HH:mm"),
+          end: moment(r.engagementDate)
+            .add(r.duration, "minutes")
+            .format("YYYY-MM-DD HH:mm"),
+          url: Report.pathFor(r),
+          classNames: ["event-" + r.state.toLowerCase()],
+          extendedProps: { ...r },
+          allDay: Settings.engagementsIncludeTimeAndDuration
```

Comment:
`?:`-expression can be simplified to `!Settings.engagementsIncludeTimeAndDuration || r.duration !== null`

Reply:
done